tests/unit_test/app/services/test_repository_service.py

```
@@ -0,0 +1,106 @@
+    def __init__(self, count=1, repos=None):
+        self._count = count
+        self._repos = repos or []
+
+    async def count_by_user(self, user_id):
+        return self._count
+
+    async def get_all_by_user(self, user_id, offset, limit):
+        return self._repos
+
+
+class StubLabelStore:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)